web/src/routes/(user)/archive/+page.svelte

```diff
@@ -0,0 +1,261 @@
+<script lang="ts">
```

Comment:
Lot of duplicate code in this file. Is there no way to reuse this between this and other pages?

Reply:
If it's not easy I'm ok having a separate PR to fix it later.